docs/developers/development-guidelines/UINewSystem.md

```diff
@@ -0,0 +1,76 @@
+# UI Refactor (March 2025)
+
+
+
+## Introduction
+
+This document contains guidelines and tutorials for Stying and Templating the UI of the application.
+
+Depending on what the control will look like or what functionality is required will determine what method is used for styling. I considering this a type of flow chart for how complex we want to go.
+
+### Styling
+
+The most basic of visual changes when functionality stays the same. Styling is when the look of a control changes without changing any underlying markup and structure. Primarily used for color and font changes for various states but is used for layout changes such as margin, padding, alignment, spacing etc.
+
+For example, if we want to style an Avalonia `Seperator` control to match our system, we would create a new `SeperatorStyles.axaml` file inside of `src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Separator`. Any created style files also need referencing in the `src/Themes/NexusMods.Themes.NexusFluentDark/Styles/StylesIndex.axaml` file.
+
+!!! info Note:
+
+    - Create a Design View for quick iteration.
+    - Use `StaticResource`'s from our design system where possible.
+
+### ControlTheme
+
+If we need more elements to style than have been provided by Avalonia's default ControlTheme, we will need to provide a new one. 
+
+For example, if we want to change the structure of a `Button` control to have a `Border` around it, we would need to create a new `ControlTheme` for the `Button` control.
+
+Copy the default `Button` control theme from the Avalonia repository into `src/Themes/NexusMods.Themes.NexusFluentDark/Controls/Button.axaml` and modify it to suit our needs.
+
+Styles and classes can be added as part of the `ControlTheme` or alternatively as a separate `Style` file. This will be determined by the complexity of the control and how many controls will use the same styles. Any created `ControlTheme`'s also need referencing in the `src/Themes/NexusMods.Themes.NexusFluentDark/Resources/ResourceIndex.axaml` file.
```

Comment:
Same as above
```suggestion
Styles and classes can be added as part of the `ControlTheme` or alternatively as a separate `Style` file. This will be determined by the complexity of the control and how many controls will use the same styles. 

Any created `ControlTheme`'s also need referencing in the `src/Themes/NexusMods.Themes.NexusFluentDark/Resources/ResourceIndex.axaml` file.
```